lib/tbd/geo.rb

```diff
@@ -372,7 +372,7 @@ def properties(model = nil, surface = nil)
       log(ERR, "Skipping '#{id}': missing construction (#{mth})")    if c.empty?
       next                                                           if c.empty?
       c = c.get.to_LayeredConstruction
-      log(ERR, "Skipping '#{id}': must be a #{cl3} (#{mth})")        if c.empty?
```

Comment:
OpenStudio subsurface constructions may de defined _inter alia_ as a ConstructionAirBoundary or using a WindowDataFile. For UA' calculations, TBD currently only extracts fenestration U-factors from LayeredConstructions. This may be revised in the future for WindowDataFiles. In the meantime, TBD skips over any subsurface construction type other than LayeredConstruction (while logging a WARNING - no longer an ERROR).